include/staff/templates/queue-tickets.tmpl.php

```diff
@@ -90,8 +90,10 @@
     $criteria->annotations = $criteria->related = $criteria->aggregated =
         $criteria->annotations = $criteria->ordering = [];
     $tickets->constraints = $tickets->extra = [];
+    $criteria->extra(array('select' => array('relevance' => 'Z1.relevance')));
     $tickets = $tickets->filter(['ticket_id__in' =>
             $criteria->values_flat('ticket_id')]);
+    $tickets->order_by(new SqlCode('A2.relevance'), QuerySet::DESC);
```

Comment:
Table aliases are auto-assigned via ORM - while they're predictable based on table relationships (joins), it's not hard-coded. So this makes me a little concerned. I don't have a solution - just raising the concern here.